src/services/util/object.js

```diff
@@ -35,3 +35,22 @@ export const unique = (array) => {
   return Array.from(new Set(array.map((el)=>{ if(isPlainObject(el)) return JSON.stringify(el); else return el}))).map(JSON.parse)
 }
 
+export const scrollToElement = (querySelector, container)=>{
+  container = container || 'body,html'
+  const qLabel = $(container).find(querySelector);
+  const qBody  = $(container);
+
+  var scrollNum = qLabel.offset().top
+
+  if(container!= 'body,html'){
+      //its a dialog calculate scrollTop
+      var dialogContainer = $(container)
+      scrollNum = scrollNum - dialogContainer.offset().top + dialogContainer.scrollTop();
+  }
+  else
+      scrollNum -= 130; //forms 
+
+  qBody.stop().animate({
+      scrollTop: scrollNum
+  }, 'slow');
+}
```

Comment:
you have this code in another PR no?

Reply:
its there in kmformworkflow PR. I have moved this code to html.js as discussed.